### PR TITLE
Screen3 ui

### PR DESCRIPTION
**Pull request by boro741**

### CODE REVIEW

File: Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart [hunk 1]
```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+class Screen3 extends StatelessWidget {
```
Comment (Roja-Vangapalli) [inline Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart]: Please rename the class name which describes it's functionality.

File: Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+              children: <Widget>[
+                RaisedButton(
+                    color: Color(0xff11b719),
+                    textColor: Colors.white,
+                    child: Padding(
+                        padding: EdgeInsets.all(10.0),
+                        child: Row(
+                          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                          children: <Widget>[
+                            Text("Proceed", style: TextStyle(fontSize: 24.0)),
```
Comment (Roja-Vangapalli) [inline Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart]: On Clicking proceed button, please show the alert dialog only if the user enter all the fields with proper values

File: Resideo_eshopping/resideo_eshopping/lib/main.dart [hunk 3]
```diff
@@ -1,111 +1,17 @@
...
-            ),
-          ],
-        ),
-      ),
-      floatingActionButton: FloatingActionButton(
-        onPressed: _incrementCounter,
-        tooltip: 'Increment',
-        child: Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
+      home: Screen3(0.0),
```
Comment (Roja-Vangapalli) [inline Resideo_eshopping/resideo_eshopping/lib/main.dart]: Please remove this code. Screen 3 will be redirected from sceeen 2. Testing code should not be pushed

Comment (boro741) [reply Resideo_eshopping/resideo_eshopping/lib/main.dart]: done

File: Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart (hunk 2 above)
Comment (boro741) [reply Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart]: done

File: Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart (hunk 1 above)
Comment (boro741) [reply Resideo_eshopping/resideo_eshopping/lib/Screens/screen3.dart]: done

**Pull request closed without merging**